rbmc-cfam-daemon/README.md

```diff
@@ -16,3 +16,18 @@ It will read the sibling's CFAM every 2 seconds. A D-Bus interface will be
 created to make these values available to other applications.
 
 The `rbmc-cfam-daemon` meson option will enable building this code.
+
+## CFAM-S ScratchPad Register Layout
```

Comment:
I decided that I'm making this readme the source of truth, so that when something changes I don't have to update the code, this readme, and some other doc.  I have an older design doc with a start at this, but I think it'd be better to have the  documentation closer to the code. 

 Does that sound OK?

Reply:
I should note that this daemon is the only user of these fields in the CFAM.

Reply:
> I decided that I'm making this readme the source of truth, so that when something changes I don't have to update the code, this readme, and some other doc. I have an older design doc with a start at this, but I think it'd be better to have the documentation closer to the code.
>
> Does that sound OK?

That makes sense to me—having a single document as the source of truth. I just want to confirm whether we're maintaining something like the figtree, which we receive from the HW/HWP procedure team. If we're solely managing the usage of these registers, then I agree with your approach of having one centralized document.